src/main/resources/camel-project-template/pom.xml

```diff
@@ -117,4 +117,108 @@
     </plugins>
   </build>
 
+<repositories>
+    <repository>
+        <id>fuse-public-repository</id>
+        <name>FuseSource Community Release Repository</name>
+        <url>https://repo.fusesource.com/nexus/content/groups/public</url>
+        <snapshots>
+            <enabled>false</enabled>
+        </snapshots>
+        <releases>
+            <enabled>true</enabled>
+            <updatePolicy>never</updatePolicy>
+        </releases>
+    </repository>
+    <repository>
+        <id>red-hat-ga-repository</id>
+        <name>Red Hat GA Repository</name>
+        <url>https://maven.repository.redhat.com/ga</url>
+        <releases>
+            <enabled>true</enabled>
+            <updatePolicy>never</updatePolicy>
+        </releases>
+        <snapshots>
+            <enabled>false</enabled>
+        </snapshots>
+    </repository>
+    <repository>
+        <releases>
+            <enabled>true</enabled>
+            <updatePolicy>never</updatePolicy>
+        </releases>
+        <snapshots>
+            <enabled>false</enabled>
+        </snapshots>
+        <id>red-hat-ea-repository</id>
+        <name>Red Hat EA Repository</name>
+        <url>https://maven.repository.redhat.com/earlyaccess/all</url>
+    </repository>
+    <!-- To remove when a long-term version will be available -->
```

Comment:
if Jboss early access repository needed? if not would be better to remove it

Reply:
Tried building without this one and it fails spectacularly.

Reply:
I am able to build without this one, but wonder if the artifacts will be moved from the http://repository.jboss.org/nexus/content/groups/ea repo into the earlyaccess repo, since building without the http://repository.jboss.org/nexus/content/groups/ea fails every time right now.

Reply:
you need to ask to Fuse runtime team